tools/container/nvidia-toolkit/run.go

```diff
@@ -114,6 +116,13 @@ func main() {
 			Destination: &options.pidFile,
 			EnvVars:     []string{"TOOLKIT_PID_FILE", "PID_FILE"},
 		},
+		&cli.StringFlag{
+			Name:        "host-root",
+			Usage:       "Specify the path to the host root to be used when executing shell commands.",
+			Value:       defaultHostRootMount,
+			Destination: &options.hostRoot,
+			EnvVars:     []string{"HOST_ROOT_MOUNT"},
+		},
```

Comment:
We don't need this here anymore, correct?

Reply:
Yes, I will remove this